pkgs/development/python-modules/openai-whisper/default.nix

```diff
@@ -43,12 +42,13 @@ buildPythonPackage rec {
     })
   ];
 
+  buildInputs = [ ffmpeg ];
```

Comment:
Why is this required? And can we use ffmpeg-headless instead? It has a much smaller closure size.

Reply:
You're right - it is not! I was confused by the release notes. I'm going to test ffmpeg-headless.

Reply:
`ffmpeg-headless`seems to work!

Reply:
Closure size reduced from 2.16 to 1.85 GiB. Over half of the rest is Pytorch.